applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -154,6 +154,7 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 
 	if (data_size > ARRAY_SIZE(iso_received->data)) {
 		ERR_CHK_MSG(-ENOMEM, "Data size too large for buffer");
+		return;
```

Comment:
I guess Coverity does not understand that ERR_CHK_MSG will cause an assert?

Reply:
Yes, seems like that.